ansible/memcached-operator/roles/Memcached/tests/test.yml

```diff
@@ -1,5 +0,0 @@
----
-- hosts: localhost
-  remote_user: root
-  roles:
-    - Memcached
```

Comment:
Why it was removed ?

Reply:
Was comparing with sample operator from USER GUIDe, and these tests were not part of that. Please let me know if these are needed in sdk samples

Reply:
Added back.

Reply:
Looks like there's still some extraneous added whitespace here.

Reply:
Fixed